Sources/LanguageServerProtocol/SupportTypes/TestItem.swift

```diff
@@ -19,6 +19,15 @@ public struct TestTag: Codable, Equatable, Sendable {
   }
 }
 
+/// An enum representing the different styles of tests that can be found in a document.
+public enum TestStyle: String, Codable, Sendable {
```

Comment:
This was previously an empty enum with static strings defined on it. I couldn't see any reason not to make it regular enum, but please let me know if there is a reason I'm not seeing.

Reply:
I wanted to define this request more generally and not restrict the test style to the two we know about in SourceKit-LSP. That’s why I kept it as an arbitrary string. With a string, there is a chance that this request could be incorporated into the LSP standard. With this enum, that wouldn’t be possible.

Reply:
I can see how we'd fail to decode an unknown TestStyle if this ever became standardized. I've reverted.